common/djangoapps/static_replace/models.py

```diff
@@ -27,3 +27,27 @@ def __repr__(self):
 
     def __unicode__(self):
         return unicode(repr(self))
+
+
+class AssetExcludedExtensionsConfig(ConfigurationModel):
+    """Configuration for the the excluded file extensions when canonicalizing static asset paths."""
+
+    class Meta(object):
+        app_label = 'static_replace'
+
+    excluded_extensions = TextField(
+        default='html',
+        help_text='The file extensions to exclude from canonicalization.  No leading period required. ' +
+        'Values should be space separated i.e. "html svg css"'
+    )
+
+    @classmethod
+    def get_excluded_extensions(cls):
+        """Gets the excluded file extensions when canonicalizing static asset paths"""
+        return cls.current().excluded_extensions.split()
```

Comment:
Should this add the `.` before the file types, to prevent accidental collisions? I honestly can't think of a real-world case where it would be a problem, but in the abstract, you wouldn't want `ml` to exclude `html` files.